src/world.rs

```diff
@@ -429,31 +447,103 @@ impl LazyInsertions {
     /// struct InsertPos;
     ///
     /// impl<'a> System<'a> for InsertPos {
-    ///     type SystemData = (Entities<'a>, Fetch<'a, LazyInsertions>);
+    ///     type SystemData = (Entities<'a>, Fetch<'a, LazyUpdate>);
     ///
     ///     fn run(&mut self, (ent, lazy): Self::SystemData) {
     ///         let a = ent.create();
     ///         let b = ent.create();
     ///
-    ///         lazy.add(vec![(a, Pos(3.0, 1.0)), (b, Pos(0.0, 4.0))]);
+    ///         lazy.insert(vec![(a, Pos(3.0, 1.0)), (b, Pos(0.0, 4.0))]);
     ///     }
     /// }
     /// ```
-    pub fn add<L>(&self, l: L)
+    pub fn insert<L>(&self, l: L)
```

Comment:
Agreed, will do that.
It would be nice if we could use execute in insert as well, to get rid of LazyInsert. But I think then FnOnce would be required, because the component would have to be moved into the closure, right?